source/sfp/1_water/sfp_114_nl_waterstaalname_plas/06_benodigdheden.Rmd

```diff
@@ -0,0 +1,113 @@
+
+-   **2 en 3**, Fles, 250 ml (**FL-250R**, **FL-250W**, **FL-250B**): vierkante fles van 250 ml met brede hals en platte bodem, vervaardigd uit high density polyethyleen (HDPE), gegradueerd, met blauwe, rode of witte schroefdop.
+
+-   **4**, Fles, 100 ml (**FL-100R**) : vierkante fles van 100 ml met smalle hals en platte bodem, vervaardigd uit high density polyethyleen (HDPE), niet-gegradueerd, met rode schroefdop.
+
+-   **5**, Falcontube, 50 ml (**FT-50B**, **FT-50R, FT-50G**): cilindrisch buisje of tube van 50 ml met conische bodem vervaardigd uit polypropyleen (PP), gegradueerd, met rode schroefdop vervaardigd uit polyethyleen (PE).
+
+-   **6**, Falcontube, 50 ml (**FT-50B**, **FT-50R, FT-50G**): cilindrisch buisje of tube van 50 ml met conische bodem vervaardigd uit polypropyleen (PP), gegradueerd, met blauwe schroefdop vervaardigd uit polyethyleen (PE).
+
+-   **7**, Falcontube, 30 ml (**FT-30W**): cilindrisch buisje of tube van 30 ml met conische bodem met rechtstaand stuk, vervaardigd uit polypropyleen (PP), niet-gegradueerd, met witte schroefdop vervaardigd uit polyethyleen (PE).
+
+![Figuur 2: De 7 recipiënten die tijdens de staalname worden gevuld.](./media/image2.jpg){width="6.299660979877515in" height="3.763888888888889in"}
+
+|               |                                    |                                                                                                |
+|:--------------|:-----------------------------------|:-----------------------------------------------------------------------------------------------|
+| **recipiënt** | **beschrijving**                   | **analyse**                                                                                    |
+| 1             | 1 l                                | zwevende stof                                                                                  |
+| 2             | 250 ml                             | chlorofyl *a*                                                                                  |
+| 3             | 250 ml                             | bicarbonaatbepaling (TAP-TAM), IC (ionenuitwisselingschromatografie), controle pH en EC, kleur |
+| 4             | 100 ml                             | TP (totaalfosfor), KjN (Kjelldahl-stikstof), COD (chemisch zuurstofverbruik)                   |
+| 5             | Falcon 50 ml rode dop              | reservestaal                                                                                   |
+| 6             | Falcon 50 ml blauwe dop            | DOC (opgelost organisch koolstof)                                                              |
+| 7             | 30 ml met 300 µl salpeterzuur (1%) | ICP (metalen)                                                                                  |
+
+Tabel 2: Analyses die uitgevoerd worden op de waterstalen van de respectievelijke recipiënten.
```

Comment:
caption definieren zodat deze bij de tabel hoort en ernaar verwezen kan worden

Reply:
Opgelost in [afb29dd](https://github.com/inbo/protocolsource/pull/107/commits/afb29ddc74b4a51c2bb60b25d47d0320220bcb39)